### PR TITLE
max amount of embeds

### PR DESCRIPTION
**Pull request by ethan-xd**

### CODE REVIEW

**Comment by DetachHead** [comment]
why do u not want to add to teh cache if theres more than 10?

**Review by shitpostbotmin — CHANGES_REQUESTED** [review]
‎

**Review by shitpostbotmin — CHANGES_REQUESTED** [review]
‎

**Comment by ethan-xd** [comment]
i that read wrong the code

File: src/main.ts [hunk 1]
```diff
@@ -57,17 +57,38 @@ const cfg = require('../config.json');
...
+	}
+
+	if (newsUpdateCount !== 0) {
+		let contentMessage = `<@&${cfg.roleID}>`;
+
+		if (newsUpdateCount > 10) {
+			contentMessage += ` (${newsUpdateCount - 10} more update${
+				newsUpdateCount - 10 !== 1 ? 's' : ''
+			} hidden)`;
+		}
```
Comment (DetachHead) [inline src/main.ts:85]: i would turn this duplicated logic into a function

**Review by DetachHead — APPROVED** [review]

**Pull request merged**